lib/assessment.md

```diff
@@ -41,7 +56,21 @@ What selectors could we write to reference each of the following categories of e
 * Any `p` elements whose parent elements are `div`s
 
 <!-- your answer starts here -->
+    ```CSS
+.big {
+  /* All elements that belong to the class `big`*/
+}
+#contentPane {
+  /* The element matching ID `contentPane` */
+}
+.important, .red {
```

Comment:
There are a number of similar-looking things in CSS that often get confused:
`.important.red`
=> Intersection of important and red (only things that are both)
`.important, .red` (note the comma)
=> Union of important and red (important things, red things, and things that are both red and important)
`.important .red`
=> Elements with the class red that are descendants (inside) of elements with the class important.